react/modules/enhancedEcommerceEvents.ts

```diff
@@ -54,7 +46,7 @@ export async function sendEnhancedEcommerceEvents(e: PixelMessage) {
         productName,
         brand,
         categories,
-      } = (e.data as ProductViewData).product
```

Comment:
With the fix of the types in `PixelMessage`, we don't need to manually cast anymore. It will correctly infer the types based on `eventName`.

<img width="620" alt="CleanShot 2023-04-25 at 20 12 25@2x" src="https://user-images.githubusercontent.com/381395/234427207-c800aef5-d8d7-4a8d-b8ea-33f2d729b960.png">